solutions/swb-reference/integration-tests/tests/multiStep/dataset.test.ts

```diff
@@ -2,85 +2,93 @@
  *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
  *  SPDX-License-Identifier: Apache-2.0
  */
-import ClientSession from '../../support/clientSession';
+// import ClientSession from '../../support/clientSession';
 import Setup from '../../support/setup';
-import RandomTextGenerator from '../../support/utils/randomTextGenerator';
-import { dsUuidRegExp } from '../../support/utils/regExpressions';
-import Settings from '../../support/utils/settings';
+// import RandomTextGenerator from '../../support/utils/randomTextGenerator';
+// import { dsUuidRegExp } from '../../support/utils/regExpressions';
+// import Settings from '../../support/utils/settings';
 
 describe('multiStep dataset integration test', () => {
   const setup: Setup = new Setup();
-  const settings: Settings = setup.getSettings();
-  let adminSession: ClientSession;
+  // const settings: Settings = setup.getSettings();
+  // let adminSession: ClientSession;
 
   beforeAll(async () => {
-    adminSession = await setup.getDefaultAdminSession();
+    // adminSession = await setup.getDefaultAdminSession();
   });
 
   afterAll(async () => {
     await setup.cleanup();
   });
 
   test('Environment provisioning with dataset', async () => {
```

Comment:
could do `test.skip` and not need to comment anything out

Reply:
neat!